docs/administration/information_schema.md

```diff
@@ -97,9 +97,11 @@ This feature is supported since StarRocks v3.0.
 
 The following fields are provided in `load_tracking_logs`:
 
-| **Field**        | **Description**                                              |
-| ---------------- | ------------------------------------------------------------ |
-| JOB_ID           | The ID of the load job.                                      |
-| LABEL            | The label of the load job.                                   |
-| DATABASE_NAME    | The database that the load job belongs to.                   |
-| TRACKING_LOG     | Error logs (if any) of the load job.                         |
+| **Field**     | **Description**                                          |
+|---------------|----------------------------------------------------------|
+| JOB_ID        | The ID of the load job.                                  |
+| LABEL         | The label of the load job.                               |
+| DATABASE_NAME | The database that the load job belongs to.               |
+| TRACKING_LOG  | Error logs (if any) of the load job.                     |
+| Type          | The type of the load job, BROKER/INSERT/ROUTINE_LOAD etc |
```

Comment:
The type of the load job. Valid values: `BROKER`, `INSERT`, and `ROUTINE_LOAD`.

And please list all the valid values if the above values are incomplete. Using "etc." in the document can be misleading. Please avoid using it.

Reply:
Thanks @EsoragotoSpirit , docs have  been updated